### DISCUSSION TITLE
4.6.32

### DISCUSSION THREAD

**Discussion by ywmoyue** [ywmoyue/biliuwp-lite#659] [Announcements]
#### ADDED
- 支持点击视频评论区时间快速空降  #640 #654 #655
- 新增收藏订阅合集页，支持查看更多订阅合集，支持取消订阅合集 #625
#### CHANGED
- 支持导入官方弹幕屏蔽文件，导入/同步弹幕屏蔽词时移除非法正则 #611
- 下载视频支持选中合集中其他视频下载，调整下载弹窗样式 #525
- 字幕描边颜色新增透明色可选，新增字幕边框背景色设置 #645
- 支持设置显示视频底部进度条 #545
- 直播间中自己弹幕添加边框，自己和主播的弹幕特殊化 #651
  - 感谢 @GD-Slime 
- 直播间中区别续费与新舰长 #651
  - 感谢 @GD-Slime 
- 直播间SC加入弹幕与chatList，天选红包信息加入屏蔽，禁言观众历史发言展示，chat栏自动滚动优化 #651
  - 感谢 @GD-Slime 
#### FIXED
- 修复多个不同合集视频在播放列表中但不会加载出第二个以上的合集 #636
- 修复视频详情页关注up主失败 #644
- 修复直播间天选抽奖点击抽奖按钮后并没有真的参与抽奖 #566 #651
  - 感谢 @GD-Slime 
- 修复某些官方直播间进入后会出错 #647 #651
  - 感谢 @GD-Slime 
- 修复直播间弹幕延迟 #651
  - 感谢 @GD-Slime 
- 修复在无直播间的用户主页还能看到直播间按钮 #651
  - 感谢 @GD-Slime 

<hr /><em>This discussion was created from the release <a href='https://github.com/ywmoyue/biliuwp-lite/releases/tag/v4.6.32'>4.6.32</a>.</em>

**Comment by Yuanmeng928** [comment]
4.6.31新添的一键播放全部视频 https://github.com/ywmoyue/biliuwp-lite/issues/576的功能，在4.6.31和4.6.32里不能正常使用，选择“全部视频”播放后，视频列表只显示了最近的20个视频

**Reply by ywmoyue** [reply]
@Yuanmeng928 好的，感谢反馈
